File: VB Examples/PierSide.vb
Module PierSide

    ' Windows Visual Basic Sample Console Application: PierSide
    '
    ' ------------------------------------------------------------------------
    '               Original Example

    '			    Developed 2015, R.McAlister
    '
    ' ------------------------------------------------------------------------
    '
    ' This Visual Basic console application reads the pier side position of the current
    '   telescope and shows the value.

    Sub Main()

        Dim tsx_tele = CreateObject("theskyx.sky6RASCOMTele")

        tsx_tele.DoCommand(11, "")
        Dim pierstring = tsx_tele.DoCommandOutput

        MsgBox("Pier Side: " + pierstring)
	Return
    End Sub

End Module

File: C# Samples/PierSide.cs
using System.Windows.Forms;
using TheSkyXLib;

public class PierSide
{

    /// Windows Visual Basic Sample Console Application: PierSide
    ///
    /// ------------------------------------------------------------------------
    ///               Original Example
    ///
    ///			    Developed 2015, R.McAlister
    ///			    Ported to C# 2017
    ///
    /// ------------------------------------------------------------------------
    ///
    /// This Visual Basic console application reads the pier side position of the current
    ///   telescope and shows the value.

    public void PierSideSample()
   {
       sky6RASCOMTele tsx_tele = new sky6RASCOMTele();
        tsx_tele.DoCommand(11, "");
        string pierstring = tsx_tele.DoCommandOutput;

        MessageBox.Show("Pier Side: " + pierstring);
	return;
   }
}